pkg/manifest/manifest.go

```diff
@@ -68,15 +70,22 @@ func (m *Manifest) Validate() error {
 	}
 	err := m.Install.Validate(m)
 	if err != nil {
-		result = multierror.Append(result, err)
+		result = multierror.Append(result, errors.Wrapf(err, fmt.Sprintf(invalidStepErrorFormat, "install")))
 	}
 
 	if m.Uninstall == nil {
 		result = multierror.Append(result, errors.New("no uninstall action defined"))
 	}
 	err = m.Uninstall.Validate(m)
 	if err != nil {
-		result = multierror.Append(result, err)
+		result = multierror.Append(result, errors.Wrapf(err, fmt.Sprintf(invalidStepErrorFormat, "uninstall")))
+	}
+
+	for actionName, steps := range m.CustomActions {
+		err := steps.Validate(m)
+		if err != nil {
+			result = multierror.Append(result, errors.Wrapf(err, fmt.Sprintf(invalidStepErrorFormat, actionName)))
+		}
```

Comment:
I added validation for custom actions also though its not relevant to issue. I think we forgot to add before

Reply:
good catch, thanks! 👍